src/nxdoc/web-ui/nuxeo-elements/nuxeo-elements-tutorial.md

```diff
@@ -191,8 +191,8 @@ Let's plug this application into the Nuxeo instance and change the hardcoded use
     # Add nuxeo-elements as a bower dependency
     $ bower install --save nuxeo/nuxeo-elements
```

Comment:
We could just update this line instead:
```suggestion
    $ bower install --save nuxeo/nuxeo-elements#2.4.x
```

Reply:
Hi Gabriel - using this install command still prompts for a version selection even after cleaning bower cache:

> Unable to find a suitable version for nuxeo-elements, please choose one by typing one of the numbers below:
    1) nuxeo-elements#2.4.62 which resolved to 2.4.62 and is required by nuxeo-ui-elements#2.4.62
    2) nuxeo-elements#2.4.x which resolved to 2.4.62
    3) nuxeo-elements#^3.0.12 which resolved to 3.0.13 and is required by polymer-starter-kit
Prefix the choice with ! to persist it to bower.json

We can still suggest this command instead, but it doesn't prevent a choice from being required. Thoughts?

Reply:
What version of `bower` are you using? I'm using the latest (`1.8.14`) and I didn't get any prompt, it installed `2.4.62`. Here's what I tested locally: `rm -rf bower_components && bower cache clean && bower install --save nuxeo/nuxeo-elements#2.4.x`.

Reply:
Nothing new on my end beyond https://github.com/nuxeo/doc.nuxeo.com-content/pull/1982#discussion_r912924773. The proposed documentation works, I just think it can be made simpler.

Reply:
Hi @Gabez0r - sorry it took a while to get back to this. I'm also using `1.8.14` and still getting prompted for the version selection, now resolving to `2.4.63`. Granted I'm testing on Windows so there may be some associated weirdness, and ultimately it does present the proper version to select on my end so I'm good to bring in the change.